yarn.lock

```diff
@@ -9,12 +9,13 @@ JSONStream@0.10.0:
     jsonparse "0.0.5"
     through ">=2.2.7 <3"
 
-accepts@~1.3.3:
-  version "1.3.3"
-  resolved "https://registry.yarnpkg.com/accepts/-/accepts-1.3.3.tgz#c3ca7434938648c3e0d9c1e328dd68b622c284ca"
+accepts@~1.3.7:
```

Comment:
This change required bumping express to latest (4.12 to 4.17), so the yarn lock will show several update since express has 30 dependencies.